src/modules/tests/tests.actions.ts

```diff
@@ -78,6 +81,15 @@ export class SendCompletedTestsFailure implements Action {
   readonly type = SEND_COMPLETED_TESTS_FAILURE;
 }
 
+export class SendPartialTestSuccess implements Action {
+  readonly type = SEND_PARTIAL_TEST_SUCCESS;
+  constructor(public testId: string) {}
```

Comment:
I think this should be names `slotId` instead. `testId` isn't really used anywhere else